flask_admin/contrib/peewee/form.py

```diff
@@ -3,6 +3,8 @@
 from peewee import (CharField, DateTimeField, DateField, TimeField,
                     PrimaryKeyField, ForeignKeyField, BaseModel)
 
+from playhouse.postgres_ext import ArrayField, JSONField, BinaryJSONField
```

Comment:
Looks like this also requires psycopg2. It's probably a good idea to handle the ImportError gracefully like:
```
try:
    from playhouse.postgres_ext import JSONField, BinaryJSONField
    has_postgres_ext = True
except:
    has_postgres_ext = False
```

```
        if has_postgres_ext:
            self.converters[JSONField] = self.handle_json
            self.converters[BinaryJSONField] = self.handle_json
```